undocked/tests/sample/ext/drv/sample_ext.c

```diff
@@ -8,19 +8,27 @@
 
 #define INITGUID
 
+// clang-format off
```

Comment:
This is a sample so should reflect what we want others to do.
So why is clang-format recommended to be off?

Reply:
I had to disable clang formatting for this file (only for the part where headers are being included), as we want to include the Windows headers _before_ ebpf header, else we get compilation errors.

Reply:
That indicates a bug in the ebpf headers.  We should fix that rather than recommending people override it.

Reply:
Fixed.